Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSource.java

```diff
@@ -83,14 +83,15 @@ private CardImageUrls innerGenerateURL(CardDownloadData card, boolean isToken) {
                 if (ScryfallImageSupportCards.isApiLink(link)) {
                     // api link - must prepare direct link
                     baseUrl = link + localizedCode + "?format=image";
-                    alternativeUrl = link + defaultCode + "?format=image";
-                    // workaround to use cards without english images (some promos or special cards)
-                    if (Objects.equals(baseUrl, alternativeUrl) && baseUrl.endsWith("/en?format=image")) {
-                        alternativeUrl = alternativeUrl.replace("/en?format=image", "?format=image");
-                    }
+                    // fallback no loc code
+                    alternativeUrl = link.substring(0,link.length()-1) + "?format=image";
                 } else {
                     // direct link to image
                     baseUrl = link;
+                    if( link.contains( "/?format=image" )){
```

Comment:
Must add comments like "workaround to use localization in direct links"

Reply:
Added workaround comments.